types/exaroton/index.d.ts

```diff
@@ -399,14 +398,14 @@ declare class Server extends EventEmitter {
      *
      * @param streams
      */
-    subscribe(streams?: subscriptionTypes[] | subscriptionTypes): boolean;
+    subscribe(streams?: subscriptionType[] | subscriptionType): boolean;
```

Comment:
Those are types (a.k.a interfaces). Would be nice, if you could take advantage and change the case (you can retail lowerCase alias for now)

Reply:
and in other places with `lowerCamelCase` notation to `UpperCamelCase`